src/main/java/dev/emi/stepheightentityattribute/mixin/EntityMixin.java

```diff
@@ -12,7 +12,7 @@
 @Mixin(Entity.class)
 public abstract class EntityMixin {
 
-	@Redirect(at = @At(value = "FIELD", target = "Lnet/minecraft/entity/Entity;stepHeight:F", opcode = Opcodes.GETFIELD), method = "adjustMovementForCollisions")
+	@Redirect(at = @At(value = "FIELD", target = "Lnet/minecraft/entity/Entity;stepHeight:F", opcode = Opcodes.GETFIELD), method = "adjustMovementForCollisions*")
```

Comment:
What is this asterisk?

Reply:
It may or may not just be an mc mod dev plugin thing, it stops it from warning about multiple methods with different parameters. I'm unsure if it's actually part of mixin, but it's functionally identical to not having it

Reply:
After looking into it it might match undesirable methods with different names in the future? I'd prefer mcdev just be allowed to complain.

Reply:
Done